data/block/metaBlock.proto

```diff
@@ -51,8 +50,6 @@ message ShardData {
 	uint64                   LastIncludedMetaNonce 				= 13 [(gogoproto.jsontag) = "lastIncludedMetaNonce"];
 	uint32                   ShardID               				= 1 [(gogoproto.jsontag) = "shardID"];
 	uint32                   TxCount               				= 7 [(gogoproto.jsontag) = "txCount"];
-	HeaderProof 						 PreviousShardHeaderProof     = 15 [(gogoproto.jsontag) = "previousHeaderProof,omitempty"];
-	uint32									 Epoch 										    = 16 [(gogoproto.jsontag) = "epoch,omitempty"];
```

Comment:
not very sure if we should still keep epoch field here, but will adapt later if needed

Reply:
we'll still need that for shard data checks and updateShardDataPeerState